File: src/main/java/cleanup/work/workcleanup/controller/form/ExcelForm.java
package cleanup.work.workcleanup.controller.form;

import lombok.Data;
import org.springframework.format.annotation.DateTimeFormat;

import java.time.LocalDate;

@Data
public class ExcelForm {

    @DateTimeFormat(pattern = "yyyy-MM-dd")
    private LocalDate startDate;
    @DateTimeFormat(pattern = "yyyy-MM-dd")
    private LocalDate endDate;

    private ExcelType excelType;        //Excel종류 (CAR, CARINSURANCE)

    private String insuranceName;       // 추출 할 보험이름
}

File: src/main/resources/static/js/excelForm.js
function submitForm() {
    const form = document.getElementById("excel_form");
    const startDate = document.getElementById("startDate");
    const endDate = document.getElementById("endDate");
    const insuranceName = document.getElementById("insuranceName");
    const excelType = document.getElementById("excelType");

    if(startDate.value == null || startDate.value == ''){
        alert("시작 날짜를 입력하세요");
        startDate.focus();
        return;
    }else if (endDate.value == null || endDate.value == ''){
        alert("끝 날짜를 입력하세요");
        endDate.focus();
        return;
    }else if ((insuranceName.value == null || insuranceName.value == '') && excelType.value == 'CAR'){
        form.submit();
    } else if(insuranceName.value == null || insuranceName.value == ''){
        alert("보험종류를 선택하세요");
        insuranceName.focus();
        return;
    }else if (excelType.value == null || excelType.value == '') {
        alert("엑셀종류를 선택하세요");
        excelType.focus();
        return;
    } else {
        form.submit();
    }
}